experimental/PropertyDDS/packages/property-dds/tsconfig.json

```diff
@@ -9,7 +9,7 @@
 		"noUnusedLocals": false,
 		"outDir": "./dist",
 		"rootDir": "./src",
-		"types": ["node"]
+        "types": ["node"]
```

Comment:
The other lines in the file use tabs: so changing this one to spaces seems bad.

Reply:
Good catch; fixed.